uwblueprint/website-bp
# Add review scoring pages

### Implementation description

- Ticket: https://www.notion.so/uwblueprintexecs/FE-Reviews-Answer-review-pages-0ea14a3ce8144cf88a877284452705e9
- Review pages implemented according to https://www.figma.com/file/hDKHlvuEcDZyPDcQRUOGV5/Review-Page?node-id=520%3A1287&mode=dev 
- Map state used in review page parent page, which is updated in each stage

<img width="1483" alt="Screenshot 2023-06-28 at 5 12 18 PM" src="https://github.com/uwblueprint/website-bp/assets/51381917/8a7a58d3-b8b9-4d7a-a43a-cf6fe236e49f">
<img width="1480" alt="Screenshot 2023-06-28 at 5 12 30 PM" src="https://github.com/uwblueprint/website-bp/assets/51381917/ec2b58fd-d809-468f-9b19-ec2daa052288">




### Steps to test

1. Start the frontend and go to `http://localhost:3000/review` (might need to sign in with blueprint account)
2. Go through the flow (can check that invalid inputs do not allow save and continue)



### What should reviewers focus on?

- State management for review scores: chose to do it this way so that we could access the scores for the EndPage

### Checklist
